src/rpc/blockchain.cpp

```diff
@@ -1924,6 +1924,37 @@ static UniValue savemempool(const JSONRPCRequest& request)
     return NullUniValue;
 }
 
+static UniValue clearmempool(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 0) {
+        throw std::runtime_error(
+            "clearmempool\n"
+            "\nRemoves all transactions from the mempool.\n"
+            "It will fail if a previous dump is still being loaded.\n"
+            "\nResult:\n"
+            "{                           (json object)\n"
+            "  \"nTx\": n,                 (numeric) The number of evicted transactions\n"
```

Comment:
Use same key as?https://github.com/bitcoin/bitcoin/blob/990e182587999dba33f2a8172cc39d4a20d2e207/src/rpc/blockchain.cpp#L1423

Reply:
Yeah, that makes sense.  (`nTx` was based on `getblock`, but `getmempoolinfo` is of course more relevant in this context.)  Will update the commit accordingly.